eZ/Publish/Core/Persistence/Legacy/Tests/User/UserHandlerTest.php

```diff
@@ -20,12 +21,12 @@
  */
 class UserHandlerTest extends TestCase
 {
-    protected function getUserHandler()
+    protected function getUserHandler(User\Gateway\DoctrineDatabase $userGateway = null)
```

Comment:
Why we did not import `User\Gateway\DoctrineDatabase`? I know we have `User\Role\Gateway\DoctrineDatabase` but the first one is duplicated several times